doc/policies/authentication.rst

```diff
@@ -391,20 +391,23 @@ type: bool
 If a user authenticates successfully all failcounter of all of his tokens
 will be reset. This can be important, if using empty PINs or *otppin=None*.
 
-always_increase_failcounter
+increase_failcounter_on_challenge
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 type: bool
 
-If a user logs in with their correct userstore password or a suitable tokenpin,
-the failcounter will still be increased for the time being.
+The normal behaviour is: to not increase the failcounter in case of challenge response.
+
+If this policy is activated the failcounter is increased for each token for which a challenge
+is triggered, even if the correct password or userstore pin is entered.
```

Comment:
The "even" is superflous. Since the correct password or pin is the requirement for triggering the challenge.

Reply:
fixed in 770f80617c7499699940c14e4b075104444da030.